Api/V8/OAuth2/Repository/AuthCodeRepository.php

```diff
@@ -0,0 +1,101 @@
+<?php
+namespace Api\V8\OAuth2\Repository;
+
+use Api\V8\BeanDecorator\BeanManager;
+use Api\V8\OAuth2\Entity\AuthCodeEntity;
+use League\OAuth2\Server\Entities\AuthCodeEntityInterface;
+use League\OAuth2\Server\Repositories\AuthCodeRepositoryInterface;
+
```

Comment:
Not required but would be nice to have PHPDocs for new classes, e.g.:

```php
/**
 * Class ClientRepository
 * @package Api\V8\OAuth2\Repository
 */
```

Reply:
Done. Also for all the other Api\V8\OAuth2\ classes.